flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_MessagesSent.js

```diff
@@ -1,9 +1,24 @@
 class CDUAocMessagesSent {
-    static ShowPage(mcdu, messages = null, offset = 5) {
+    static ShowPage(mcdu, messages = null, page = 0) {
         if (!messages) {
             messages = mcdu.atsuManager.aoc.outputMessages();
         }
         mcdu.clearDisplay();
+        mcdu.page.Current = mcdu.page.AOCSentMsgs;
+
+        page = Math.min(Math.floor((messages.length - 1) / 5), page);
```

Comment:
`page = Math.max(0, Math.min(Math.floor((messages.length - 1) / 5), page));`

Reply:
@tracernz 
Made this change - testing it during the day.
Did you look at my other changes as well? I'm a bit concerned as this PR is a bit larger as I anticipated and needs good review and testing.

Reply:
Looks good to me. One more QA pass to merge (since it failed for ShadowFrost)?

Reply:
Yes, maybe two :) it is an important component